addons/binding/org.openhab.binding.sensebox/ESH-INF/thing/box.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sensebox" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- This thing represents the Arduino box that sends data to api.opensensebox.org . -->
+	<thing-type id="box">
+		<label>Sensebox</label>
+		<description>Sensor Box</description>
+
+		<channel-groups>
+			<channel-group typeId="descriptors" id="descriptors"/>
```

Comment:
If I am not mistaken, I cannot use "properties" in the Paper UI Control tab to visualize  data. Neither can I bind them to Items to use in rules or sitemaps. Therefore, I would like to keep the "descriptors" as channels. @kaikreuzer ?